superset-frontend/src/setup/setupApp.ts

```diff
@@ -33,7 +33,7 @@ function showApiMessage(resp: ClientErrorObject) {
   const severity = resp.severity || 'info';
   $(template)
     .addClass(`alert-${severity}`)
-    .append(resp.message || '')
+    .append((resp.message || '') as string)
```

Comment:
instead of coercing, could we do a better check here? 

I presume `resp.message` could be a variety of different types, perhaps it would be better to do `String(resp.message)`? Or maybe `resp.message ? String(resp.message) : ''`